README.md

```diff
@@ -21,3 +21,65 @@ While testing against a real kafka has many benefits for integration testing, it
 Ktest is backed by TopologyTestDriver and certainly is still in need of performance improvements but regardless of that it is very easy to get setup with and start running quick tests.
 
 Ontop of that, ktest provides a way to introduce deterministic randomness, you can opt in to deterministically shuffle the inputs to a topology before processing to catch those timing issues you might not have expected.
+
+## Build & Test
+
+The library has a few java classes in the original kafka namespaces, mostly just to provide public accessors to internal methods (although `CapturingStreamTask` also provides some functionality). As a result, it can't be run in a Clojure repl until the java classes have been build.
+
+To build the java classes (and run the tests), run the `test.sh` script. Once the java classes have been built, you can run the clojure code and tests in a repl as normal, although of course any change to the java classes will require you to rerun the script and restart your repl.
```

Comment:
can mention here that to build the java classes they can run `clj -M:build`, otherwise I can imagine people running test all the time just to build them

Reply:
Done.